### DISCUSSION TITLE
How to stop upload to PVOUTPUT and only publish MQTT payloads

### DISCUSSION THREAD

**Discussion by fritserasmus** [SBFspot/sbfspot-config#63] [Q&A]
I have a RPi to ONLY send the readings to my MQTT broker reading. SBF-Spot connects through the IP addresses of my two SBs
It works quite well, BUT it also tries to send values to PVOUTPUT.

1. From time to time the RPi misses some readings and I think it is because it attempts to connect to PVOUTPUT.
Where and how can I stop that upload? 

2. That said, SBF-Spot stopped sending payloads to my MQTT server and I do not know where to start with the investigation.
It is quite evident that based on the previous question, the logfile has a zillion entries for: 
`ERROR: Uploading 30 datapoints, starting with 20230503,07:00,33239316,0,,,20.1492,230.79 Unauthorized 401: Invalid API Key`
and nothing else.

The last entries in the CSV file created are perfectly fine, but the last entry is more than an hour ago

I would really appreciate it if I could get help with this.

Kind Regard

Frits

**Comment by sbf-** [comment]
> It works quite well, BUT it also tries to send values to PVOUTPUT.

SBFspot doesn't handle uploads to PVOutput. This is done by SBFspotUpload daemon.

> Where and how can I stop that upload?

`sudo systemctl disable SBFspotUpload.service`

> SBFspot stopped sending payloads to my MQTT server and I do not know where to start with the investigation

Enable logging with `-v5` on the SBFspot command line (See `/usr/local/bin/sbfspot.3/daydata` script)
Log files are stored in `/var/log/sbfspot.3`

**Reply by fritserasmus** [reply]
Thank you sbf,

I had to redo a re-install as it turned out the SD card got corrupted.

The interface to install all the variables is absolutely FANTASTIC!!! 

Thanks for that effort.

I  selected eth parameters NOT to feed to PVOUTPUT and not to save to an .csv file so my SD card should last much longer this time!!!

Regards
Frits
